cernopendata_client/cli.py

```diff
@@ -243,6 +243,13 @@ def get_file_locations(server, recid, doi, title, protocol, expand, verbose):
         DOWNLOAD_RETRY_SLEEP
     ),
 )
+@click.option(
+    "--download-lib",
+    "dl_lib",
```

Comment:
`dl_lib` is not very descriptive. What about keeping the same variable name as the CLI option name. This will make searchers etc easier.

`--download-lib` would be better spelled `--download-library`. But if we keep this option also for XRootD and any other future methods, what about simply calling it `--downloader`? Or perhaps `--downloader-engine`?

Reply:
`--downloader-engine` sounds good to me.